.babelrc

```diff
@@ -1,5 +1,10 @@
 {
-  "presets": ["es2015", "stage-2"],
-  "plugins": ["transform-runtime"],
+  "presets": [
+    ["@babel/env", {
+      "targets": {
+        "browsers": ["last 2 versions", "IE 11"]
```

Comment:
Let's add a `.browserslistrc` file, it's a better way than keep it just for babel, b/c, for example, Autoprefixer can't use it.